src/main/java/com/github/mc1arke/sonarqube/plugin/ce/pullrequest/azuredevops/model/GitPullRequestStatus.java

```diff
@@ -0,0 +1,26 @@
+package com.github.mc1arke.sonarqube.plugin.ce.pullrequest.azuredevops.model;
+
+import com.github.mc1arke.sonarqube.plugin.ce.pullrequest.azuredevops.model.enums.GitStatusState;
+
+public class GitPullRequestStatus {
+    /**
+     * ID of the iteration to associate status with. Minimum value is 1.
+     */
+    public Integer iterationId = null;
```

Comment:
These should be private, and access through getters, with initialisation in the constructor

Reply:
Resolve